proto/mls/message_contents/credential.proto

```diff
@@ -8,15 +8,6 @@ import "mls/message_contents/association.proto";
 option go_package = "github.com/xmtp/proto/v3/go/mls/message_contents";
 option java_package = "org.xmtp.proto.mls.message.contents";
 
-// A credential that can be used in MLS leaf nodes
```

Comment:
If we're putting this into main, let's just leave this old credential as-is and add the new one. That way it's not a breaking change

Reply:
I rebased this to `identity` branch, deleted part is for CI to pass.

Reply:
I still think we might want to keep the old one around for a bit. Once we start using this in `libxmtp` we are going to have to update every single reference to the old credential immediately or compiles will fail.